bundle_sync/main.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"reflect"
+	"sort"
+	"strings"
+	"time"
+
+	cfg "github.com/RedHatInsights/entitlements-api-go/config"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	endpoints = []string{"ansible", "smart_management", "openshift_container_platform"}
```

Comment:
I think instead of this `endpoints` variable, we could just reference the config here: https://github.com/RedHatInsights/entitlements-api-go/blob/master/config/main.go#L130